### DISCUSSION TITLE
Welcome to OSPP-2022 Discussions!

### DISCUSSION THREAD

**Discussion by faweizhao26** [kubesphere/community#411] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of SOPP-2022. We hope that you:
  * Ask questions you’re wondering about the project of OSPP-2022.
  * Show your progress of your project.
  * Share ideas.
  * Engage with other  members.

  So you can comment below with your questions or progress or ideas.

**Comment by YADROOKIE** [comment]
OpenFunction Node.js Framework (WW27)

1. Completed the Kubernetes & OpenFunction installation on QingCloud's VM
1. Development of plugin system for sync runtime: ongoing ~35%

**Reply by YADROOKIE** [reply]
OpenFunction Node.js Framework (WW29)

1. Development of plugin system  (sync & async) : ongoing ~ 70%

**Reply by YADROOKIE** [reply]
1.  Completed async function  plugin system  [pr](https://github.com/OpenFunction/functions-framework-nodejs/pull/70)
2. Start  async function skywalking plugin  [pr](https://github.com/OpenFunction/functions-framework-nodejs/pull/74)

**Comment by jairuigou** [comment]
Kubeeye Console

1. Setup KubeSphere and KubeEye environment on Cloud.
2. Review component design with ChenZhen.
3. Component style development in progress.

**Reply by jairuigou** [reply]
Completed component styles and interaction logic.

**Comment by chengleqi** [comment]
FluxCD Integration

We use [issue](https://github.com/kubesphere/ks-devops/issues/716) to track the progress.

**Comment by 77yu77** [comment]
KubeKey Phase Run:
1.artifact import   [issue](https://github.com/kubesphere/kubekey/issues/1389)   and  [pr](https://github.com/kubesphere/kubekey/pull/1398)
2.upgrade binary   [issue](https://github.com/kubesphere/kubekey/issues/1417)  and   [pr](https://github.com/kubesphere/kubekey/pull/1418)
3. image [pr](https://github.com/kubesphere/kubekey/pull/1425)
4. cluster nodes [pr](https://github.com/kubesphere/kubekey/pull/1430)
5. ks upgrade [pr](https://github.com/kubesphere/kubekey/pull/1441)
